src/evergreen/task.py

```diff
@@ -111,7 +111,6 @@ class Task(_BaseEvergreenObject):
     display_only = evg_attrib("display_only")
     distro_id = evg_attrib("distro_id")
     est_wait_to_start_ms = evg_attrib("est_wait_to_start_ms")
-    estimated_cost = evg_attrib("estimated_cost")
```

Comment:
Removed this attribute on `Task.py`. This prevents autocompletion in editors for `estimated_cost`. This will discourage users from using this field.